src/Geocodio.php

```diff
@@ -180,13 +183,15 @@ private function handleException(Exception $e) {
         throw new GeocodioException($errorMessage, $errorCode, $e);
     }
 
-    private function formatResponse(ResponseInterface $response) {
-        return json_decode((string)$response->getBody());
+    private function formatResponse(ResponseInterface $response): mixed
+    {
+        return json_decode((string) $response->getBody());
     }
 
-    private function getHeaders(): array {
+    private function getHeaders(): array
+    {
         return [
-            'User-Agent' => 'geocodio-library-php/1.2.0'
+            'User-Agent' => 'geocodio-library-php/1.2.0',
```

Comment:
I guess this is fine since it is addressed in #11 instead

Reply:
Yeah, we address this in #11